data/mods/aftershock_exoplanet/items/armor/cyberpunk.json

```
@@ -12,6 +12,24 @@
     "material_thickness": 0.3,
     "pocket_data": [  ]
   },
+  {
+    "id": "afs_mirror_shades",
+    "repairs_like": "glasses_bal",
+    "type": "ITEM",
+    "subtypes": [ "ARMOR" ],
+    "copy-from": "glasses_bal",
+    "looks_like": "glasses_bal",
+    "name": { "str": "mirror shades", "str_pl": "pairs of mirror shades" },
+    "description": "A pair of mirrored combat sunglasses.  They look pretty badass, but they give you a definitive disadvantage when it comes to playing cards."
```

I dunno I just dont like the word badass haha.
```suggestion
    "description": "A pair of mirrored combat sunglasses.  An essential piece of streetwear, they confer a definitive disadvantage when it comes to playing cards."
```

---

You're the boss o7